docs/admin/templates/managing-templates/devcontainers/add-devcontainer.md

```diff
@@ -0,0 +1,143 @@
+# Add a dev container template to Coder
+
+A Coder administrator adds a dev container-compatible template to Coder
+(Envbuilder). This allows the template to prompt for the developer for their dev
+container repository's URL as a
+[parameter](../../extending-templates/parameters.md) when they create their
+workspace. Envbuilder clones the repo and builds a container from the
+`devcontainer.json` specified in the repo.
+
+You can create template files through the Coder dashboard, CLI, or you can
+choose a template from the
+[Coder registry](https://registry.coder.com/templates?tag=devcontainer):
+
+<div class="tabs">
+
+## Dashboard
+
+1. In the Coder dashboard, select **Templates** then **Create Template**.
+1. To use a
+   [starter template](https://github.com/coder/coder/tree/main/examples/templates),
+   select **Choose a starter template**.
+
+   - Select **From scratch** to create a new template and enter information and
+     file contents manually.
+
+## CLI
+
+1. Use the `template init` command to initialize your choice of image:
+
+   ```shell
+   coder template init --id devcontainer-kubernetes
+   ```
+
+   A list of available templates is shown in the
+   [templates_init](../../../../reference/cli/templates.md) reference.
+
+1. `cd` into the directory and push the template to your Coder deployment:
+
+   ```shell
+   cd $_ && coder templates push
```

Comment:
https://asciinema.org/a/EJW6rx6KmgkWSLiLE0yutfUZ6

Reply:
this might've been a problem with my setup + possibly something oh-my-zsh handles without my realizing:

<https://asciinema.org/a/KkdTlsBIQqrnCFIqkR6Wv8Tcw>

Reply:
thanks! this might be an issue with my setup - I think oh-my-zsh handled this for me without my realizing

https://asciinema.org/a/KkdTlsBIQqrnCFIqkR6Wv8Tcw